app/storybook/documentation/ui-kit/QueryHooks.mdx

```diff
@@ -0,0 +1,60 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
+import LinkTo from '@storybook/addon-links/react'
+import { SourceCode } from '../../.storybook/blocks/SourceCode'
+import * as useCounterQueryStories from '../../../../packages/ui-kit/src/hooks/useCounter/useCounter.stories'
+
+import * as useLeaderboardStories from '../../../../packages/ui-kit/src/hooks/useLeaderboard/useLeaderboard.stories'
+
+import * as useTimeSeriesStories from '../../../../packages/ui-kit/src/hooks/useTimeSeries/useTimeSeries.stories'
+
+<Meta title="Customization/Query Hooks" />
+
+# Query Hooks
+
+Propel’s UI Kit library provides the following React components: Counter, TimeSeries, and Leaderboard. To query the GraphQL API, we have developed 
+Query Hooks including "useCounter", "useLeaderboard" and "useTimeSeries.
+
+## useCounter
+
+<SourceCode of={useCounterQueryStories.FractionUnicode} shown />
+
+### `CounterQueryProps`
+
+See the documentation below for a complete reference for CounterQueryProps
+
+- <LinkTo kind="API/CounterQueryProps" story="overview">
+    CounterQueryProps
+  </LinkTo>
+
+## useTimeSeries
+
+<SourceCode of={useTimeSeriesStories.TimeSeriesChart} shown />
+
+### `TimeSeriesQueryProps`
+
+See the documentation below for a complete reference for TimeSeriesQueryProps
```

Comment:
Since we mentioned `TimeSeriesQueryProps` in backticks within the heading, should we also mention it using backticks below? What is the style we are following? We should perhaps also have a trailing colon (":"):
```suggestion
See the documentation below for a complete reference for `TimeSeriesQueryProps`:
```

Reply:
I think we can ommit the ### `CounterQueryProps` sub heading in backticks.
SO, I update it/